site/templates/siteComponent.pug

```diff
@@ -68,7 +68,7 @@ html(lang='en-US' dir="ltr").spectrum.spectrum--light.spectrum--medium
 
               .spectrum-CSSComponent-switcher
                 div.spectrum-CSS-switcherContainer
-                  label.spectrum-FieldLabel.spectrum-FieldLabel--left(for='switcher-theme') Theme
+                  label.spectrum-FieldLabel.spectrum-FieldLabel--sizeM(for='switcher-theme') Theme
```

Comment:
@pfulton Reminder for you & I to make sure this update makes it into the refactor work @Rajdeepc has done.

Reply:
@castastrophe we can do this now, actually. It's lookin' mighty weird on the production site, and I caught it with these updates.

Reply:
Sorry for the confusion, I didn't mean we don't make this update but rather to remember to migrate this update in the *.njk files b/c raj already rewrote them all.